geowebcache/core/src/main/java/org/geowebcache/storage/BlobStoreListenerList.java

```diff
@@ -27,88 +27,88 @@ public synchronized boolean removeListener(BlobStoreListener listener) {
     }
 
     public void sendLayerDeleted(String layerName) {
-        if (listeners.size() > 0) {
-            for (int i = 0; i < listeners.size(); i++) {
-                listeners.get(i).layerDeleted(layerName);
-            }
-        }
+        listeners.stream().forEachOrdered(listener->{
```

Comment:
Seems me a 

````
listeners.forEach(listener -> layerDeleted(layerName));
````

would have been at the same time more compact, readable, and faster (streams are very slow)